src/taskgraph/util/dependencies.py

```diff
@@ -27,6 +27,11 @@ def group_by_single(config, tasks):
         yield [task]
 
 
+@group_by("all")
+def group_by_all(config, tasks):
+    return [[task for task in tasks]]
```

Comment:
Shouldn't this just be `return [tasks]`?

Reply:
That was my instinct as well, but nope. [The transform](https://github.com/taskcluster/taskgraph/blob/main/src/taskgraph/transforms/from_deps.py#L151-L158) pretty clearly wants a list of lists.

(`group_by_attribute` also returns a list of lists. And `group_by_single` yields a list in each iteration of its loop.)